### ISSUE TITLE
kube-bench version command does not work in docker image

### ISSUE THREAD

**Issue by ttousai** [aquasecurity/kube-bench#391]
kube-bench version: aquasec/kube-bench:latest
sha256: 719706c60ed5a08051c335d7654c65f4ab312957a2aec7805acbf4965794b556

Running the version command in the docker image for aquasec/kube-bench returns nothing rather than the version of kube-bench.

Reproduce:
```
$ sudo docker pull aquasec/kube-bench@sha256:719706c60ed5a08051c335d7654c65f4ab312957a2aec7805acbf4965794b556
$ sudo docker run aquasec/kube-bench:latest version
```

**Comment by robertojrojas** [comment]
Fixed by #428

**Comment by robertojrojas** [comment]
This issue is back or probably I didn't fix it correctly :(

**Comment by robertojrojas** [comment]
Fixed by #522
